packages/sdk/src/restclient.ts

```diff
@@ -362,11 +358,26 @@ export class RestClient {
     return unwrapWasmResponse(responseData) || [];
   }
 
-  // throws error if no contract at this address
-  public async getContractInfo(address: string): Promise<ContractDetails> {
+  /**
+   * Returns null when contract was not found at this address.
+   */
```

Comment:
If this is desired behavior, i can chango the server to return null not error. Please make an issue on wasmd

Reply:
The reason I want to handle the _not found_ case explicitely is to allow setting a nice custom error message in the `CosmWasmClient`, since I suspect this case to be relatively common.

I think the cleanest solution for a missing resource in `GET /wasm/contract/${address}` is to return HTTP 404 with whatever content. That would avoid the need to look into error texts alltogether. See https://github.com/cosmwasm/wasmd/issues/89